cmd/arcaflow/main.go

```diff
@@ -197,31 +200,36 @@ func runWorkflow(flow engine.WorkflowEngine, fileCtx loadfile.FileCache, workflo
 		cancel()
 	}()
 
-	workflow, err := flow.Parse(fileCtx, workflowFile)
+	workFlowObj, err := flow.Parse(fileCtx, workflowFile)
 	if err != nil {
 		logger.Errorf("Invalid workflow (%v)", err)
 		return ExitCodeInvalidData
 	}
 
-	outputID, outputData, outputError, err := workflow.Run(ctx, inputData)
-	if err != nil {
-		logger.Errorf("Workflow execution failed (%v)", err)
-		return ExitCodeWorkflowFailed
-	}
-	data, err := yaml.Marshal(
-		map[string]any{
-			"output_id":   outputID,
-			"output_data": outputData,
-		},
-	)
-	if err != nil {
-		logger.Errorf("Failed to marshal output (%v)", err)
-		return ExitCodeInvalidData
-	}
-	_, _ = os.Stdout.Write(data)
-	if outputError {
-		return ExitCodeWorkflowErrorOutput
+	if getNamespaces {
+		workflow.PrintNamespaceResponse(os.Stdout, workFlowObj.Namespaces(), logger)
```

Comment:
Maybe `PrintNamespaceReport` (or "Table") would be better than "Response"? Or even `ReportNamespaces`?

Also, if you added a `return` here, you could avoid the `else` and changing the indent of the lines below. The function already has multiple returns, so that would help rather than hurt readability.

Reply:
I changed it to `PrintObjectNamespaceTable`.

Reply:
Added the `return`